docs/docs-content/clusters/edge/site-deployment/site-installation/cluster-deployment.md

```diff
@@ -55,11 +51,26 @@ Use the following steps to create a new host cluster so that you can add Edge ho
 
 9. The node configuration page is where you can specify what Edge hosts make up the host cluster. Assign Edge hosts to the **master-pool** and the **worker-pool**. When you have completed configuring the node pools, click on **Next**.
 
+10. (Optional) When you assign Edge hosts to node pools, you can optionally specify a static IP address for each Edge host. If you want to specify a static IP, toggle on **Static IP** and provide the following information:
 
-10. The Settings page is where you can configure a patching schedule, security scans, backup settings, and set up Role-Based Access Control (RBAC). Review the settings and make changes if needed. Click on **Validate**.
+  | **Field** | **Description** |
+  |-------|-------------|
+  | IP Address | The static IP address assigned to your Edge host. This should be unique within your network. |
+  | Default gateway | The IP address of the default gateway for your cluster network. This gateway routes traffic from your cluster to external networks, including the internet. |
```

Comment:
```suggestion
  | Default gateway | The IP address of the default gateway for your cluster network. This gateway routes traffic from your cluster to external networks. |
```
Let's remove the internet statement. Internet access is an assumption that could cause concerns. In reality, this is controlled by the customer's network configuration.